wp-content/themes/pub/wporg-learn-2024/patterns/sensei-lesson-header.php

```diff
@@ -14,11 +14,7 @@
 	<!-- wp:group {"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space","top":"0px","bottom":"0px"}}},"className":"sensei-course-theme-header-content","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
 	<div class="wp-block-group sensei-course-theme-header-content" style="padding-top:0px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:0px;padding-left:var(--wp--preset--spacing--edge-space)">
 
-		<!-- wp:group {"className":"wporg-breadcrumbs","align":"full","style":{"spacing":{"padding":{"top":"23px","bottom":"23px","right":"var:preset|spacing|edge-space"}}},"backgroundColor":"white","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
-		<div class="wporg-breadcrumbs wp-block-group alignfull has-white-background-color has-background" style="padding-top:23px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:23px;">
-			<!-- wp:wporg/site-breadcrumbs {"fontSize":"small"} /-->
-		</div>
-		<!-- /wp:group -->
+		<!-- wp:pattern {"slug":"wporg-learn-2024/site-breadcrumbs"} /-->
```

Comment:
Seem to have gained extra padding here

![Screenshot 2024-07-04 at 4 08 39 PM](https://github.com/WordPress/Learn/assets/1017872/786c48b2-07dd-47e0-998f-a40156c73089)

Reply:
Fixed in 0888452